include/users.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+    function getAllUsers() {
+        $users = dbQuery("
+            SELECT *
+            FROM users
+        ")->fetchAll();
+        return $users;
+    }
+    
+    function getUser($userId) {
+        $user = dbQuery("
+            SELECT *
+            FROM users
+            WHERE userId = $userId
```

Comment:
I forget the exact syntax, but I think you want to escape this somehow. Did you cover that in any of the initial lessons? Usually it would have something like a `:` in front of it here, but maybe that hasn't been covered yet!